### DISCUSSION TITLE
[GUI] Set Ok as default option in prompts

### DISCUSSION THREAD

**Discussion by bombillazo** [dbeaver/dbeaver#20085] [Q&A]
Hello, is there any way to set the default of all prompts to the OK option instead of the Cancel option?

![image](https://github.com/dbeaver/dbeaver/assets/44002321/bbb32ab2-66a3-4ec2-8b7d-b43efcf0915f)

**Comment by LonwoLonwo** [comment]
Hello. 
Nope, we do not have such an option. Because this is can be a danger for our users.
Options that you can change for your required value you can find in Preferences -> User Interface -> Confirmations

**Reply by bombillazo** [reply]
Ok thanks for confirming.
